### ISSUE TITLE
Request for Colab Pro for Students

### ISSUE THREAD

**Issue by SupreethRao99** [googlecolab/colabtools#2367]
Hello,

It would be great if Colab Pro or Colab Pro + could be offered to students in the following ways
1. At a discounted rate OR
2. For a specified number of hours a week after which it would default to a non Pro version of colab.

 It would really help us learn and experiment.  

Thank you !

**Comment by hackedXD** [comment]
What does triaged mean, is Colab Pro available for students? Frankly it would be a very useful tool, especially combined with github development pack.

**Comment by JovanXin** [comment]
Agreed, this would be very helpful

**Comment by shyamal-anadkat** [comment]
+1. following. This would be a blessing

**Comment by Muhammad0Awais** [comment]
+1. following. Looking forward to it.

**Comment by CorpaciLC** [comment]
+ 1 agreed

**Comment by juliafalcao** [comment]
+1 agreed!

**Comment by mohammadreza490** [comment]
+1 This will definitely help us a lot

**Comment by kirilldolmatov** [comment]
+1 absolutely agreed

**Comment by imloualvaro** [comment]
upvote

**Comment by wuhjie** [comment]
+1 absolutely agreed!

**Comment by shreyshrivastava4799** [comment]
+1 we need this!

**Comment by mogokhalifa** [comment]
+1 agreed!

**Comment by imnublet** [comment]
I agree! +1

**Comment by interittus16** [comment]
+1 agreed!

**Comment by AlexSperka** [comment]
+1 agree

**Comment by rajdeepmondal1** [comment]
+1 Agree

**Comment by egbakou** [comment]
+1 Agree
I am about to email google asking for discount or something like that :)

**Comment by Tarandeep97** [comment]
+1 Agreed

**Comment by C4r4c0l3** [comment]
+1

**Comment by alfinauzikri** [comment]
+1 Agree

**Comment by Matsumotorise** [comment]
+1

**Comment by nathan-149** [comment]
+1 <3

**Comment by artifyr** [comment]
+1

**Comment by EvelinaAlexiutenko** [comment]
+1

**Comment by slobodaapl** [comment]
+1 please

**Comment by Sairam954** [comment]
+1

**Comment by AdelDima** [comment]
+1

**Comment by abecaduo** [comment]
+1

**Comment by jesus-333** [comment]
+1

**Comment by matuszewskis** [comment]
+1 agree

**Comment by zabdulz** [comment]
+1

**Comment by SmallSquare** [comment]
+1 agree

**Comment by 19521242bao** [comment]
+1 agree

**Comment by Mob-Nikhil898** [comment]
+1 agree

**Comment by haydentinker** [comment]
+1 agree

**Comment by bqbnakasprv7aqe6e** [comment]
+1 big agree

**Comment by ayoubSMO** [comment]
Agree !

**Comment by jordivv** [comment]
+1

**Comment by Mohabeldiin** [comment]
+1

**Comment by rayhendricks** [comment]
+1 agree!

**Comment by runnily** [comment]
+1

**Comment by aalvan** [comment]
+1

**Comment by dberweger2017** [comment]
+1

**Comment by Quanta-of-solitude** [comment]
+1

**Comment by Nicialy** [comment]
+1

**Comment by Oct4Pie** [comment]
Would be quite helpful

**Comment by ewaldkleefstra** [comment]
+1

**Comment by ahsan-raza** [comment]
+1

**Comment by haha0205** [comment]
It would be helpful

**Comment by Mawesomes** [comment]
+ 2

**Comment by leleogere** [comment]
+1

**Comment by CoKhiBachKhoa** [comment]
+1

**Comment by argishh** [comment]
Much Needed!

**Comment by PhilipMathieu** [comment]
+1

**Comment by ShanemelAsuncion** [comment]
+1

**Comment by GhezaeGoitom** [comment]
Agree!

**Comment by yash-02** [comment]
> Hello,
> 
> It would be great if Colab Pro or Colab Pro + could be offered to students in the following ways
> 
> 1. At a discounted rate OR
> 2. For a specified number of hours a week after which it would default to a non Pro version of colab.
> 
> It would really help us learn and experiment.
> 
> Thank you !
Agree!

**Comment by yash-02** [comment]
Agree!

**Comment by hen3kr8** [comment]
+1

**Comment by YangBarry** [comment]
+1

**Comment by Wetiqe** [comment]
+1

**Comment by erikzimmermann** [comment]
+1

**Comment by ivictoruan** [comment]
+ 1

**Comment by MultiWu** [comment]
+1

**Comment by ParmarKrishna** [comment]
+1

**Comment by maxicastro** [comment]
+1

**Comment by timothygao8710** [comment]
🙏

**Comment by rawdhikagupta** [comment]
+1 that will be really helpful

**Comment by ynes99** [comment]
+1

**Comment by MishaRubanov** [comment]
+1, would help a lot!

**Comment by PradoVarathan** [comment]
+1

**Comment by williamberrios** [comment]
+1

**Comment by shi-ivam** [comment]
+1

**Comment by EfeSenerr** [comment]
+1

**Comment by Midoran** [comment]
1+

**Comment by KARTHIK-KG** [comment]
+ 1

**Comment by dorzhey** [comment]
+1

**Comment by nanda-kumar-k** [comment]
+1 that will be really helpful.

**Comment by meliy-meyada** [comment]
+1 agreed!

**Comment by GuilhermeGraeff** [comment]
+1 agreed!

**Comment by shreyasd2301** [comment]
+1

**Comment by muhammadanas365** [comment]
+1

**Comment by Canahmetozguven** [comment]
+1

**Comment by Olabiyisam** [comment]
+1

**Comment by GoodLuck95** [comment]
+1

**Comment by tanishq14developer** [comment]
+1

**Comment by vornao** [comment]
+1

**Comment by babakmbm** [comment]
+1

**Comment by klil21** [comment]
+1

**Comment by sandeepKBM** [comment]
+1

**Comment by JadonLee8** [comment]
+1 agreed!

**Comment by yyaprak** [comment]
+1

**Comment by martingrzzler** [comment]
+1

**Comment by skurnapally** [comment]
+1

**Comment by rhobro** [comment]
Is this going to go forward?

**Comment by sarisemih** [comment]
+1

**Comment by KevinArce** [comment]
+ 1

**Comment by notcooler** [comment]
+1

**Comment by Taichi-Pink** [comment]
+1

**Comment by koytipita** [comment]
+1

**Comment by miguelgargallo** [comment]
+1

**Comment by Hrushal-Nikhare** [comment]
+1

**Comment by vritansh** [comment]
+1

**Comment by AhmedNasser1601** [comment]
Agree +1

**Comment by kinivi** [comment]
+1

**Comment by umar052001** [comment]
+1 Agreed

**Comment by sam-the-programmer** [comment]
+1 i think this would be amazing!!!

**Comment by safiaafrin** [comment]
+1 Agreed

**Comment by Mohamed-Badr** [comment]
+1

**Comment by ammarlodhi255** [comment]
+1 Strongly agreed

**Comment by hmiche** [comment]
Agreeeeeeeeee

**Comment by whoisraibolt** [comment]
+1

**Comment by sam-the-programmer** [comment]
Any more progress on this???

**Comment by sainishwanth** [comment]
A discount for students would honestly be a great option

**Comment by AymenSe** [comment]
Especially in the third world, we don't even have a credit cards. 
we cannot pay for it, so could you please offer it to students with an academic email?

**Comment by bereniceluis** [comment]
+1

**Comment by laxmandunde** [comment]
+1

**Comment by tmatzxzone** [comment]
+1

**Comment by alghozi06** [comment]
+1

**Comment by HakemShuabr** [comment]
+1

**Comment by msshahzaib777** [comment]
+1

**Comment by Lukas1h** [comment]
+1000

**Comment by sam-the-programmer** [comment]
+100000000000

We need this. Please. 🙏

**Comment by ashfaqsasalatti** [comment]
+1

**Comment by MatiasColazo** [comment]
+1

**Comment by whtitefall** [comment]
+1

**Comment by vale-sch** [comment]
+1

**Comment by Patatone** [comment]
+1

**Comment by corbanvilla** [comment]
+1

**Comment by alfo93** [comment]
+1

**Comment by filippobartolucci** [comment]
+1

**Comment by sherryahy** [comment]
+1

**Comment by hanniamed** [comment]
+1

**Comment by HeMuling** [comment]
+1

**Comment by AfnanCSE98** [comment]
+1
indeed it would be really helpful

**Comment by varshapwalia** [comment]
+1

**Comment by PhilipMathieu** [comment]
+1

**Comment by Hannes0730** [comment]
+1
any updates on this?

**Comment by ardulat** [comment]
+1

**Comment by Dattito** [comment]
+1

**Comment by riskykrnawan** [comment]
+1 agreed!

**Comment by Daniellop1** [comment]
+1

**Comment by vannezha** [comment]
+1

**Comment by islamsaeed9854** [comment]
+1

**Comment by nicofretti** [comment]
+1

**Comment by lukik45** [comment]
+1

**Comment by malibayram91** [comment]
+1 Agreed!

**Comment by chaitanya2593** [comment]
+1 agreed

**Comment by QuwsarOhi** [comment]
+1

**Comment by Miyamura80** [comment]
+1

**Comment by DheemanthBhat** [comment]
+1

**Comment by congdinhchi** [comment]
+1

**Comment by MohammedZ666** [comment]
+1

**Comment by shkhaksar** [comment]
+1

**Comment by antoniyaaboyanova** [comment]
+1

**Comment by elhambuxton** [comment]
Agreed +1

**Comment by nothintohide** [comment]
+1

**Comment by josephinoo** [comment]
+1

**Comment by adelovelace** [comment]
Agreed +1

**Comment by nikhil-shrestha** [comment]
+1

**Comment by zhenwilliam** [comment]
I agree too. But it seems that no such student plan has been being mulled over yet.

**Comment by Habibu-R-ahman** [comment]
+1

**Comment by Ataago** [comment]
+1

**Comment by aarush-dev** [comment]
+1 It would be really helpful

**Comment by Raphael-Aguiar** [comment]
+ 1

**Comment by givinghawk** [comment]
+1 would be very helpful for students like me, just trying to get into AI!

**Comment by abhishektandon** [comment]
Should be afforable for researchers

**Comment by akramex-dz** [comment]
upvote (☞ﾟヮﾟ)☞

**Comment by Mahdi-Hasan** [comment]
+1

**Comment by onurravli** [comment]
It would be great.

**Comment by w1redch4d** [comment]
+1

**Comment by QQRyan** [comment]
+1

**Comment by Aeyxen** [comment]
That will be great.

**Comment by hephaestus88** [comment]
+1

**Comment by SamariddinS** [comment]
+1

**Comment by Necrozma200** [comment]
+1

**Comment by skyepurchase** [comment]
+1

**Comment by carlcrede** [comment]
+1

**Comment by redha-me** [comment]
+1

**Comment by DervlexArt** [comment]
1+

**Comment by githuberuser** [comment]
Totally agree

**Comment by fe7ng654123** [comment]
+1

**Comment by missflash** [comment]
+1

**Comment by Shalvi-Saxena** [comment]
Totally agree!

**Comment by Hassanjk** [comment]
+1

**Comment by Pritam-hakingmaster** [comment]
+1

**Comment by MantautasRimkus** [comment]
+1

**Comment by Billy-06** [comment]
+1

**Comment by amiraBENMOU** [comment]
+1 it will help us a lot in our master  degree project  or phd's

**Comment by YuenSzeHong** [comment]
agree, thought i have only 1 year to graduate

**Comment by slokeshrestha26** [comment]
+1 would be helpful

**Comment by EmileBigot** [comment]
+2

**Comment by waron97** [comment]
+1

**Comment by maxkonrad** [comment]
+1

**Comment by gusadammu** [comment]
+1

**Comment by mlimbuu** [comment]
+1

**Comment by mlimbuu** [comment]
+1

**Comment by m0saan** [comment]
+1

**Comment by huuthocs** [comment]
+1

**Comment by daksh95** [comment]
+1

**Comment by coolhazersex** [comment]
eee

**Comment by nayse** [comment]
+1

**Comment by VavRe** [comment]
+1

**Comment by SuryaDhanasekaran** [comment]
+1 agreed

**Comment by zeel-04** [comment]
+1 Agree

**Comment by fakhrist** [comment]
+1 Agree, it will help a lot of us as a student or educational institution 🙏

**Comment by dev-mirzabicer** [comment]
following

**Comment by NXKED** [comment]
+1 this

**Comment by adi-panda** [comment]
+1

**Comment by rabbitjojo** [comment]
yes, please!

**Comment by AsrieltheGoat** [comment]
+1

**Comment by Patrickliuu** [comment]
+1

**Comment by TommasoCere** [comment]
+1

**Comment by alongoldenberg** [comment]
+1

**Comment by gustavobartho** [comment]
+1

**Comment by tellihichem** [comment]
+1

**Comment by Anuj-malviya0** [comment]
+1

**Comment by Young1906** [comment]
+1

**Comment by 5c0rp264** [comment]
+1

**Comment by halitsever** [comment]
👀

**Comment by Tongue923** [comment]
+1

**Comment by ACprime96** [comment]
+!

**Comment by Imran1221** [comment]
+1

**Comment by suryakantkf** [comment]
++

**Comment by mazambaloch7** [comment]
> Thanks for the suggestion!

Please think about helping the student. it would be helpful if you made a free plan for students.  please

**Comment by mazambaloch7** [comment]
My request is to make the colab pro completely free for students. Like Microsoft helps students, in many ways. with copilot and other ways

**Comment by adelovelace** [comment]
+1 agreed!

**Comment by g-lopes** [comment]
+1, that would be great!

**Comment by djthegr8** [comment]
+1 agreed

**Comment by DUMBANIKET** [comment]
+1

**Comment by riccihenrique** [comment]
+1

**Comment by mackevinbraza** [comment]
+1

**Comment by joaolucasete** [comment]
+1

**Comment by Benalla-Aiman** [comment]
+1 agreed

**Comment by rony0000013** [comment]
+1

**Comment by LeanderPolzin** [comment]
+1

**Comment by ez2rok** [comment]
This would be really helpful!

**Comment by THRINADH43** [comment]
+!
Agreed

**Comment by usernein** [comment]
+1
Please!

**Comment by edwinjosechittilappilly** [comment]
+1 Please
Providing Colab Pro Plus for students and PhD researchers would be a valuable initiative to support their academic and scientific endeavors. Colab Pro Plus can be a useful tool for enhancing the quality and productivity of education and research.

**Comment by martin-creator** [comment]
I completely agree with this request! As a fellow student, I've faced similar challenges during my projects. Adding Colab Pro to the GitHub Student Pack would be a tremendous boost for all of us, enabling more efficient training and empowering us to achieve greater heights in our research. Let's support this proposal!

**Comment by alirezanobakht13** [comment]
++

**Comment by JethroNatividad** [comment]
+1

**Comment by kato1628** [comment]
+1

**Comment by Muk74dir** [comment]
+1

**Comment by ilyasozdogan** [comment]
+1

**Comment by suganda8** [comment]
Having Google Colab in student's sleeve would be really helpful for learning & research. however let's keep cope.

**Comment by oleks-andr** [comment]
It would be great to have additional free resources for students. There are many tasks, especially those related to artificial intelligence and machine learning, that require a lot of GPU resources.

**Comment by animeshsg** [comment]
+1

**Comment by CharlesDDDD** [comment]
+10086

**Comment by tsantosh1098** [comment]
+1

**Comment by benjaminorr** [comment]
+1

**Comment by guangmeainuan** [comment]
+1

**Comment by YamacTan** [comment]
+1. Local workstations will never be enough for MSc and PhD studies.

**Comment by BobChrismansyah** [comment]
+1

**Comment by robertprp** [comment]
+1

**Comment by krll-corp** [comment]
+1. Fully agree, it's worth it for the students to use the remote hardware while they don't have to buy the equipment. Furthermore, in some tasks that require a lot of RAM, the Colab Pro is (for me) the best solution. For example, some my University tasks require training AI models, and the best solution I found is Colab

**Comment by sejal0328** [comment]
+1

**Comment by ISlothI** [comment]
+1

**Comment by maruthi-syn-ack** [comment]
Indeed We need it now more than ever it changes a lot

**Comment by raincoast** [comment]
Does it support now?

**Comment by harshshah-codes** [comment]
+1 Agreed

**Comment by Mohamed-Hamza** [comment]
+1 yes please!

**Comment by PhelaPoscam** [comment]
+1

**Comment by yunzinan** [comment]
+1

**Comment by HachiroSan** [comment]
+1

**Comment by SamirXR** [comment]
DEFINATELY!

**Comment by migellebarlis** [comment]
+1

**Comment by kaden-young** [comment]
+1

**Comment by mjahmed-wd** [comment]
+1

**Comment by calza3000** [comment]
+1

**Comment by RustyGrackle** [comment]
+1 we need this

**Comment by Gunjan000** [comment]
+1

**Comment by Ricardogn224** [comment]
+1

**Comment by SuperDim666** [comment]
+1 
![image](https://github.com/googlecolab/colabtools/assets/52674623/ab82f267-4e09-4497-8bf6-b709dd6bb106)

**Comment by masheransari** [comment]
+1

**Comment by All3yp** [comment]
+1 🥺

**Comment by GabrielaBezerra** [comment]
+1 😭😭😭

**Comment by Tanujshriyan** [comment]
+1 Give it give it! I need it badly!

**Comment by kenza-benkirane** [comment]
+1

**Comment by thlurte** [comment]
+1

**Comment by 1varunvc** [comment]
+1

**Comment by iamthehimansh** [comment]
+1

**Comment by asad0x01** [comment]
+∞

**Comment by amr552** [comment]
+1 yes, please. it will be great and save the students time

**Comment by danielhotze** [comment]
+1, would really help for my Machine Learning exercises at univerity

**Comment by hoangtrung1801** [comment]
+1 really helpful

**Comment by rclarsfull** [comment]
+1

**Comment by Champzzz** [comment]
+1

**Comment by Devesh-Maheshwari** [comment]
+1 yes, please. it will be great

**Comment by fareed23** [comment]
2️⃣
Yes please, we would love it!

**Comment by i-harshit-sharma** [comment]
+1

**Comment by attia-mahmoud** [comment]
+1

**Comment by u-green** [comment]
+1

**Comment by vuyelwadr** [comment]
+1

**Comment by AlejandroDaneri** [comment]
+1

**Comment by dori654** [comment]
+1

**Comment by markushaug** [comment]
+1

**Comment by sreeniSudharsan** [comment]
+1 Still waiting on this 🥺

**Comment by MoAbbasid** [comment]
+1, its been 3 years google, help the students

**Comment by matinmahmoudi** [comment]
+1

**Comment by StGorazd** [comment]
+1

**Comment by sadanddan** [comment]
+1

**Comment by taneya1987** [comment]
I can really make use of this for my coursework right now.
Google colab pro should be free for university students. atleast for the masters and phd ones.

**Comment by lynkos** [comment]
**+1** This would be indescribably helpful for (1) [student] researchers and (2) university students of all levels: Undergrad, masters, and PhD students doing their coursework, capstone project, senior design project, thesis/dissertation, research paper(s), etc.

**Comment by karkir0003** [comment]
> Thanks for the suggestion!

@cperry-goog any update on this request? What would the ETA be on the team's roadmap?

**Comment by AnwarHawash1** [comment]
**+∞**

**Comment by gmijo47** [comment]
+1

**Comment by amiraBENMOU** [comment]
sure ,we will have more fun exploring machine learning and creating new models

**Comment by doduythao** [comment]
+1

**Comment by iu-bb** [comment]
+1

**Comment by karkir0003** [comment]
I submitted this feedback along with links to the GitHub issue in the feedback form page within Colab. Hopefully 🙏 the team takes a look and adds it to their roadmap

**Comment by iu-bb** [comment]
Until approval and implementation, students become professional 😂

**Comment by iu-bb** [comment]
not yet, and honestly i Don't expect student helping stuff from Google.
In another case google provide some free stuff to students in google cloud,
you see the process to get approved? to annoying, hard and like a s**t for
normal student. same thing in google startup forms.

**Comment by meiob** [comment]
100% Please include this in GitHub Education pack!

**Comment by iu-bb** [comment]
google will never do this, specifically collaboration with Student
Developer Pack (my GitHub, powered by Microsoft haha. 😂

I mentioned whole story in previous comment.

**Comment by A-Khadija** [comment]
hey guys does anyone have google coolab pro ?

**Comment by cperry-goog** [comment]
Hi folks, thanks for chiming in here, we've been working on this for a long time and finally have an update to share.

Colab Pro and Pro+ have launched today for Workspace organizations, giving Workspace customers (like schools and universities) the ability to buy Colab Pro and Pro+ at discounted rates for longer commits: https://workspaceupdates.googleblog.com/2024/06/google-workspace-colab-pro-and-colab-pro-plus.html

It is my hope that more students can access Colab Pro with this offering.

In this update, there is no change to our offering that is free of charge - you can continue to use Colab as before.

Please let me know if you have feedback. I imagine this will only address some of the hopes expressed in this thread, so keeping it open.

**Comment by DUMBANIKET** [comment]
I think many universities aren't using google workspace so , maybe adding an option for availing this by edu accounts would be much better.
Though nice improvement 😄

**Comment by Wicky2001** [comment]
+1

**Comment by rajputrv** [comment]
Absolutely colab pro + must be available for students fro free of cost

**Comment by GWX-101** [comment]
yes, maybe a few hours every week of colab pro would be really helpful

**Comment by Diganto002** [comment]
+1

**Comment by LuisJoseSanchez** [comment]
+1

**Comment by kazemihabib** [comment]
+1

**Comment by nushankodikara** [comment]
+1

**Comment by OmuNaman** [comment]
+1, Need For Individual Students ;-; (Especially For High Grads)

**Comment by SongTonyLi** [comment]
+ 1

**Comment by gihandjay** [comment]
+1

**Comment by philsv** [comment]
+1

**Comment by krupallad09** [comment]
+1 agreed

**Comment by Dannynis** [comment]
+1

**Comment by MuhammadMesum514** [comment]
+!

**Comment by louiscklaw** [comment]
+1

**Comment by seifelwarwary** [comment]
+1

**Comment by sbarcelona11** [comment]
+1

**Comment by emmaboccaletti** [comment]
discounted prices of google colab would be amazing, so we could validate through our educational institutions.

**Comment by glisca** [comment]
+1

**Comment by smsnobin77** [comment]
+1

**Comment by yGifoom** [comment]
+1

**Comment by LI3ARA** [comment]
We need this

**Comment by nascetti-a** [comment]
+ 1

**Comment by Akash-nath29** [comment]
+1, it would be so useful to have colab pro as a student

**Comment by Yifang-Guo** [comment]
+1

**Comment by O7Furkan17** [comment]
+1

**Comment by shreesha345** [comment]
+1

**Comment by Aaryan-Kapoor** [comment]
+1

**Comment by aydnIbrahim** [comment]
+1

**Comment by Unzerror** [comment]
+1

**Comment by pccassin** [comment]
+1

**Comment by Syedshajib55** [comment]
+1

**Comment by nesclass** [comment]
+1

**Comment by m-muaz** [comment]
+1

**Comment by samueltomen** [comment]
+1

**Comment by GKAPOOR17** [comment]
+1 It would be nice to have colab with extended features

**Comment by Si7li** [comment]
+1

**Comment by masterfelu** [comment]
+1

**Comment by EricChiu147** [comment]
+1

**Comment by chidaksh** [comment]
+1

**Comment by redezvous2004** [comment]
+1

**Comment by pwguimar** [comment]
+1

**Comment by navaneethreddydevops** [comment]
+1

**Comment by Shahubzg** [comment]
+1

**Comment by aldookware** [comment]
+1

**Comment by beenycool** [comment]
+1
but i dont think theyre going to imple,ment it

**Comment by fraadap** [comment]
+1!

**Comment by Vidya-0119** [comment]
+1

**Comment by mu21043** [comment]
+1

**Comment by antonrejoe** [comment]
+1

**Comment by usmanahmad11** [comment]
+1 Agreed.

**Comment by dennohpeter** [comment]
+3

**Comment by Cihaimasuiro** [comment]
+ 1

**Comment by Cihaimasuiro** [comment]
+1

**Comment by hiroshil** [comment]
available now: [Colab Pro for Education](https://colab.research.google.com/signup)
detail -> [New Google Colab features for higher education](https://blog.google/outreach-initiatives/education/colab-higher-education/)


<img width="1920" height="1080" alt="Image" src="https://github.com/user-attachments/assets/ecd3961f-f0b7-41cc-b9af-000c3ce68745" />

**Comment by judemars** [comment]
Hi from the Colab team! Yes, as @hiroshil mentioned, we have released a limited number of no cost Pro subscriptions to eligible, US-based students and faculty. Read more in [our announcement blog](https://blog.google/outreach-initiatives/education/colab-higher-education/), signup [here](https://colab.research.google.com/signup), and [see our FAQ](https://research.google.com/colaboratory/faq.html#edu-faq-heading) for more technical questions!

**Comment by OmuNaman** [comment]
Sad :(

**Comment by Tehada** [comment]
Hi, thanks for this great option. On my edu account it shows None in top right corner and I can't apply for it (US based University):

<img width="1512" height="982" alt="Image" src="https://github.com/user-attachments/assets/18fb4345-3180-4de9-a061-e30e05963609" />

**Comment by judemars** [comment]
@Tehada are you signed in with your Google account? Also, what happens when you click "No cost for students and educators"?

**Comment by Tehada** [comment]
> [@Tehada](https://github.com/Tehada) are you signed in with your Google account? Also, what happens when you click "No cost for students and educators"?

It was a problem from my side -- my account had disabled these services (Colab particularly) by default so I had to enable them and then it started to work.

**Comment by judemars** [comment]
Glad to hear it's working now!

**Comment by Tehada** [comment]
@judemars sorry but I have another problem now🙈 I've uploaded my signed offer for PhD position but system still rejects it. Can you help me with this?

<img width="1057" height="831" alt="Image" src="https://github.com/user-attachments/assets/8db6cd38-c7c3-45b7-b33e-30f45c1335a6" />

**Comment by judemars** [comment]
You'll need to reach out to SheerID for help with the verification process. Here's our FAQ on how to do that: https://research.google.com/colaboratory/faq.html#edu-verification-questions.
